Dockerfile

```diff
@@ -40,7 +41,7 @@ COPY --from=helm-go-lib-builder /helm-go-lib-build/helm-scala-sdk/helm-go-lib /l
 # Install the Helm3 CLI client using a provided script because installing it via the RHEL package managing didn't work
 RUN curl -fsSL -o get_helm.sh https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3 && \
     chmod 700 get_helm.sh && \
-    ./get_helm.sh --version v3.2.4 && \
+    ./get_helm.sh --version v3.11.2 && \
```

Comment:
Bumped helm version in Leo -- was needed for `oci` chart, and probably good to do anyway. (App deploys seem to work still, including Galaxy)